target/linux/ipq806x/patches-4.19/0055-cpufreq-dt-Add-L2-frequency-scaling-support.patch

```diff
@@ -43,6 +43,13 @@ Signed-off-by: Georgi Djakov <georgi.djakov@linaro.org>
 +			l2_freq = clk_get_rate(l2_clk);
 +
 +			if (l2_freq != new_l2_freq) {
++				/* set l2_freq to idle before setting target l2_freq - there seems 
```

Comment:
This belongs to commit description, but please remove this patch if its not needed to get 4.19 working.